### PR TITLE
DOT Maven Plug-in experiment

### PR DESCRIPTION
**Pull request by atopper**
## Description

Addition of experiment to play with and learn about the DOT Maven Plug-in.

## Related Issue

https://jira.corp.adobe.com/browse/CEL-152

## Motivation and Context

Making it easier to learn how to use the DOT to make Dispatcher configurations better.

## How Has This Been Tested?

Yes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: experiments/optimizer/README.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
+# Using the Dispatcher Optimizer Tool (DOT)
+
+This experiment demonstrates how to use the Dispatcher Optimizer Tool, its Maven Plug-In in particular, to create
+better configurations, following best practices and easily spotting inadvertent configuration errors.
+
+## Compatibility
+
+This experiment is compatible with both AEM 6.5 and AEM as a Cloud Service.
+
+## Problem
```
Comment (clotton) [inline experiments/optimizer/README.md:10]: This Problem section is really good!

File: experiments/optimizer/README.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+* The DOT is used in Adobe's AEM Cloud Service Pipeline, where it is run automatically to analyze the contained
+  configuration.
+* A developer can use the open-source DOT Core library to create a new application, perhaps for the command line.
+
+## Setup
+
+Follow the instructions on the main experiments page.
+
+***
+![#ff0101](https://via.placeholder.com/15/ff0101/000000?text=+)
```
Comment (clotton) [inline experiments/optimizer/README.md:33]: Just checking if this placeholder url is ok to be here?

File: experiments/optimizer/README.md [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+- Save the file and run the DOT again.
+- Check the "results.html" file and see if the `DOTRules:Disp-7` appears.
+
+### Create a rule
+
+Your organization may come up with its own rules.  Let's say you decide that /filter's with "ALLOW" should not be
+permitted to specify the value using a /glob setting.  Let's implement that.
+
+- Edit `dispatcher-config-basic/rules/experiment_rules.json`
+- Delete its contents and replace it the following:
```
Comment (clotton) [inline experiments/optimizer/README.md]: missing "with" in this sentence.

**Review by clotton — APPROVED** [review]
couple minor comments

File: experiments/optimizer/README.md (hunk 2 above)
Comment (atopper) [reply experiments/optimizer/README.md:33]: Ya?  It is a trick to add color to the README.  Kinda weird though.

**Pull request merged**